tests/unit_tests/simtel/test_simtel_runner_camera_efficiency.py

```diff
@@ -82,8 +86,28 @@ def test_check_run_result(simtel_runner_camera_efficiency):
         simtel_runner_camera_efficiency._check_run_result()
 
 
-def test_get_one_dim_distribution(simtel_runner_camera_efficiency):
-    camera_filter_file = simtel_runner_camera_efficiency._get_one_dim_distribution(
+def test_get_one_dim_distribution(site_model_south, simtel_path, telescope_model_sst_prod5):
+
+    logger.warning(
+        "Running test_get_one_dim_distribution using prod5 model "
+        " (prod6 model with 1D transmission function)"
+    )
+
+    # 2D transmission window not defined in prod6; required prod5 runner
```

Comment:
Maybe adding TODO here to include this later, or 2D is not planned for prod6?

Reply:
This I don't know - and we also need to separate more (much more) model versions from testing code. What we want to test here is if the simtel_runner_camera_efficiency is working . This is done.

I have tried to generally use prod6 for testing as the model is more verified in terms of schema. As the prod6 model is fixed, I don't expect that there will be a 2D transmission window added.